ui/apps/platform/src/Containers/Clusters/DelegateScanning/DelegateScanningPage.tsx

```diff
@@ -100,15 +184,14 @@ function DelegateScanningPage() {
                 </Flex>
             </PageSection>
             <PageSection>
-                {!!errMessage && (
+                {!!alertObj && (
                     <Alert
-                        title="Problem retrieving the delegated scanning configuration from the server"
-                        variant={AlertVariant.danger}
+                        title={alertObj.title}
```

Comment:
Seeing `title` without `body` above reminded me to ask if you can see if axe DevTools finds a problem with heading levels? If it does, `Alert` has a `component` to specify appropriate heading element for the page.

Reply:
Indeed! it needed to be h2 in this context.